apps/metadata-editor-e2e/src/e2e/dashboard.cy.ts

```diff
@@ -313,6 +313,43 @@ describe('dashboard (authenticated)', () => {
       })
     })
   })
+  describe('search filters', () => {
+    describe('allRecords search filter', () => {
+      beforeEach(() => {
+        cy.visit('/catalog/search')
+      })
+      it('should filter the record list by editor (Barbara Roberts)', () => {
+        cy.get('md-editor-search-filters').find('gn-ui-button').first().click()
+        cy.get('.cdk-overlay-container')
+          .find('input[type="checkbox"]')
+          .eq(1)
+          .check()
+        cy.get('gn-ui-interactive-table')
+          .find('[data-cy="table-row"]')
+          .should('have.length', '5')
+        cy.get('gn-ui-results-table')
+          .find('[data-cy="table-row"]')
+          .first()
+          .as('record')
+        cy.get('@record')
+          .children('div')
+          .eq(4)
```

Comment:
This getting a Nth child of an element always feels a bit brittle to me. Can we easily add a data-cy attribute on the needed element here? There are other examples of data-cy in the code.

Reply:
Yep, I've adapted accordingly. It didn't save that many lines, but has the advantage to easier test all the entries and not only the first :-)